package.json

```diff
@@ -41,16 +41,14 @@
     ".gitignore"
   ],
   "dependencies": {
-    "loader-utils": "^1.1.0"
-  },
-  "peerDependencies": {
-    "papaparse": "^4.5.0"
+    "loader-utils": "^1.1.0",
+    "papaparse": "^5.1.0"
   },
+  "peerDependencies": {},
```

Comment:
Since there are no more peer dependencies, you can just delete this

Reply:
Sure. Will do